.github/workflows/cargo-tests.yml

```diff
@@ -57,6 +57,15 @@ jobs:
       - name: install rust toolchain
         uses: actions-rust-lang/setup-rust-toolchain@v1
 
+      - name: Install toml-sort
+        run: cargo install --git https://github.com/4meta5/toml_sort
```

Comment:
I don't like the dependencies to the personal repositories. We may leave it as it is right now, but lets consider publishing your tool.

Reply:
Oooooh
I see.
Then the your approach is correct.
Lets merge it as is and consider two options:
* pushing Moonsong to merge the fix
* taking ownership over the `toml-sort` on OpenZeppelin's behalf 

By the way, we may progress with both things at the same time. Forking to the OpenZeppelin repo and creating an issue along with the fix in the PR back to Moonsong. While it is not merged, we are using our stable git dependency in OZ fork, as it gets merged, we are switching back to the main one.